orchagent/portsorch.cpp

```diff
@@ -49,6 +52,9 @@ PortsOrch::PortsOrch(DBConnector *db, vector<string> tableNames) :
     /* Initialize port table */
     m_portTable = unique_ptr<Table>(new Table(m_db, APP_PORT_TABLE_NAME));
 
+    /* Initialize queue table */
+    m_queueTable = unique_ptr<Table>(new Table(counter_db, COUNTERS_QUEUE_NAME_MAP));
```

Comment:
try make_unique?
If it works, please help modify existing code above.

Reply:
Why do we need to mix refactoring with new functionality? This line is written in the same style as above. If there's a need for this kind of change, it should be done separately.